OPTIONS.md

```diff
@@ -0,0 +1,34 @@
+## VarTable Options
+
+
+
+Usage
+---------
+
+    $ vartable_report --input-vcf [../filename.vcf] --type [basecaller] \
+		--input-bam [../filename.bam] --input-ref [../filename.fasta] \
+		--minpercent [20] --minbq [25] --mindepth [10] \
+		--output-err [filename.err] --input-prm [../filename.fasta] \
+		--output-name [filename.tsv]
+
+## (Required)
+**--input-vcf**		input VCF file
+
+**--type**		input flag (base_caller | lofreq), this differentiates b/t a VCF with all positions and VCF with only variant positions
+
+**--input-bam**		input BAM file
+
+**--input-ref**		input reference FASTA file or GenBank, gb extension file (this will parse for annotated information)
+
```

Comment:
Besides accepting the reference FASTA for this flag, I thought it'd be a good option to input a GenBank file, in case, the reference had annotations, such as the UTR and CDS regions. Papa's comment below got me thinking of another option. Option --target-cds-region would allow the user to optionally input a target region instead, so only variants in that region would be parsed into an output file. Do you think this would be useful?

I think given the right circumstances it could be useful. I did a HAdV vaccine strain analysis last year, where I isolated one nt variant of interest and compared that position to all HAdVs found on GenBank. The result was that the variant was not observed in nature; only some vaccine strains showed this variation. If I had this feature, it would have saved me some time.